Python/Product/VSInterpreters/Interpreter/CondaEnvironmentFactoryProvider.cs

```diff
@@ -343,7 +403,7 @@ internal class CondaInfoResult {
             }
 
             var arch = CPythonInterpreterFactoryProvider.ArchitectureFromExe(interpreterPath);
-            var version = CPythonInterpreterFactoryProvider.VersionFromSysVersionInfo(interpreterPath);
+            var version = await Task.Run(() => ExecuteCondaListAsync(condaPath, name));
```

Comment:
It seems that `ExecudeCondaListAsync` is always wrapped in `Task.Run` to ensure that it runs on a background thread. It might be better to move that logic into the method itself - basically just use `ConfigureAwait(false)` for all its awaits.

Reply:
I believe there's only one await in that function. So you're saying just `await ExecuteCondaListAsync`, then use ConfigureAwait(false) inside the method?

Reply:
Yep, that should do the trick.

Side note: we should probably get into the habit of using `ConfigureAwait(false)` in general, except for those places where a postback to the original thread is actually needed. The default in .NET is really unfortunate - it brings a significant perf hit and increases possibility of deadlocks. But, this might be tricky for existing code, where it might not be clear whether any given code block that follows an `await` implicitly relies on it being the UI thread or not. So it might be best to do this piecemeal whenever we significantly modify the code, and have to ask those questions anyway.